tests/integration/static_ip_address_test.yaml

```diff
@@ -0,0 +1,463 @@
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  ipv6:
+    enabled: true
+    dhcp: false
+    autoconf: false
+    address:
+    - ip: 2001:db8:1::1
+      prefix-length: 64
+description: Configure the ethernet device eth2 to have the address 2001:db8:1::1/64
+interfaces:
+- name: eth2
+  type: ethernet
+  state: up
+  ipv6:
+    enabled: true
+    dhcp: false
+    autoconf: false
+    address:
+    - ip: 2001:db8:1::1
+      prefix-length: 64
+description: Configure the ethernet device eth1 to have the address 2001:0db8:85a3:0000:0000:8a2e:0370:7331/64
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  ipv6:
+    enabled: true
+    dhcp: false
+    autoconf: false
+    address:
+    - ip: 2001:db8:85a3::8a2e:370:7331
+      prefix-length: 64
+description: Configure the eth1 ethernet device with address 192.0.2.251/24, configure the eth2 ethernet device with address 192.0.2.251/24
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: false
+    address:
+    - ip: 192.0.2.251
+      prefix-length: 24
+- name: eth2
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: false
+    address:
+    - ip: 192.0.2.251
+      prefix-length: 24
+description: Configure the ethernet device eth1 with the address 2001:db8:1::1/64, configure the ethernet device eth2 with the address 2001:db8:1::1/64
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  ipv6:
+    enabled: true
+    dhcp: false
+    autoconf: false
+    address:
+    - ip: 2001:db8:1::1
+      prefix-length: 64
+- name: eth2
+  type: ethernet
+  state: up
+  ipv6:
+    enabled: true
+    dhcp: false
+    autoconf: false
+    address:
+    - ip: 2001:db8:1::1
+      prefix-length: 64
+description: Configure the eth1 ethernet device with the address 198.51.100.249/24
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: false
+    address:
+    - ip: 198.51.100.249
+      prefix-length: 24
+description: Configure the eth2 ethernet device with address 198.51.100.250/24
+interfaces:
+- name: eth2
+  type: ethernet
+  state: up
+  ipv4:
+    enabled: true
+    dhcp: false
+    address:
+    - ip: 198.51.100.250
+      prefix-length: 24
+description: Configure the eth1 ethernet device with the address 2001:db8:1::1/64 and 2001:db8:2::1/64
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  ipv6:
+    enabled: true
+    dhcp: false
+    autoconf: false
+    address:
+    - ip: 2001:db8:2::1
+      prefix-length: 64
+    - ip: 2001:db8:1::1
+      prefix-length: 64
+description: Configure the eth1 ethernet device with address 2001:db8:1::1/64
+interfaces:
+- name: eth1
+  type: ethernet
+  state: up
+  ipv6:
+    enabled: true
+    dhcp: false
+    autoconf: false
+    address:
+    - ip: 2001:db8:1::1
+      prefix-length: 64
+description: Configure the eth1 ethernet device with IPv6 enabled
```

Comment:
This is not disabling IPv6 but with empty IPv6 address. User will still get a IPv6 link local address.

So maybe `with IPv6 enabled with link local address only`.